cmake/Dependencies.cmake

```diff
@@ -1421,6 +1419,19 @@ if(USE_DISTRIBUTED AND USE_TENSORPIPE)
   endif()
 endif()
 
+if(USE_DISTRIBUTED AND USE_UCC)
+  if(USE_SYSTEM_UCC)
+    find_package(UCC)
+    find_package(UCX)
```

Comment:
so this works as long as UCX & UCC's lib directories are specified on LD_LIBRARY_PATH?

Reply:
CMake find_package uses the following search procedure: https://cmake.org/cmake/help/latest/command/find_package.html#search-procedure
If UCX/UCC is installed in a non-standard prefix, you can set the following variable to let cmake search for that prefix
https://cmake.org/cmake/help/latest/variable/CMAKE_PREFIX_PATH.html#variable:CMAKE_PREFIX_PATH